index.html

```diff
@@ -0,0 +1,190 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="author" content="Andrii Kravchenko">
+    <title>Form</title>
+</head>
+<body>
+    <header>
+        <div>
+            <a href="#">
+                <img src="img/LOGO.png" alt=Logo>
+            </a>
+            <div>
+                <nav>
+                    <ul>
+                        <li><a href="">Company</a></li>
+                        <li><a href="">Services</a></li>
+                        <li><a href="">FinTech Solutions</a></li>
+                        <li><a href="">Products</a></li>
+                        <li><a href="">Portfolio</a></li>
+                        <li><a href="">Contact us</a></li>
+                    </ul>
+                </nav>
+            </div>
+        </div>
+        <div>
+            <a href="#">
+                <img src="img/search.png" alt="search">
+            </a>
+        </div>
+        <div>     
+            <a href="#">
+                <img src="img/notification.png" alt="search">
+            </a>
+        </div>
+        <button>
+            <img src="img/Avatar.png" alt="Avatar">
+        </button>
+    </header>
+    <main> 
+        <div>
+            <h1>Contact us</h1>
+            <strong>Need an experienced and skilled hand with custom IT projects?<br>Fill out the form to get a free consultation.</strong>
+        </div>
+        <div>
+            <form>
+                <p>Your company name*</p>
+                <input type="text">
+                <p>Nature of Business*</p>
+                <input type="text">
+                <p>Data of registration</p>
+                <input type="date" name="" id="">
+            </form>
+        </div>
+        <div>
+            <form>
+                <p>Country of registration*</p>
+                <select name="">

```

Comment:
name? id? value? All of them must be filled in
Please add some options for different countries to make it look real.
https://www.w3schools.com/tags/tag_select.asp

Reply:
+ validation